packages/tenant-process/src/routers/TenantRouter.ts

```diff
@@ -512,6 +513,33 @@ const tenantsRouter = (
           return res.status(errorRes.status).json(errorRes).end();
         }
       }
+    )
+    .delete(
+      "/tenants/:tenantId/attributes/certified/:attributeId",
+      authorizationMiddleware([ADMIN_ROLE, M2M_ROLE]),
+      async (req, res) => {
+        const ctx = fromAppContext(req.ctx);
+        try {
+          const { tenantId, attributeId } = req.params;
+          await tenantService.revokeCertifiedAttributeById(
+            {
+              tenantId: unsafeBrandId(tenantId),
+              attributeId: unsafeBrandId(attributeId),
+              organizationId: req.ctx.authData.organizationId,
+              correlationId: req.ctx.correlationId,
+            },
+            ctx.logger
```

Comment:
```suggestion
            },
            ctx
```
You already have everything in `ctx`, no need to pass the others too